spec/routing/cases_routes_spec.rb

```diff
@@ -13,86 +13,86 @@
 
     context 'manager user' do
       let(:user) { manager }
-      it { should redirect_to '/cases/open' }
+      xit { should redirect_to '/cases/open' }
```

Comment:
I'm a bit uncomfortable marking these specs as pending without more explanation as to why and a JIRA ticket to address it later. When we say, "it is now irrelevant", is that a temporary thing? Can we delete these specs instead?

Reply:
Good spot - resolved by re-enabling and updating the routing. Separated out the 'filters' controller routes into a separate file

Reply:
Goo spot - resolved